miscellaneous/build_templates/analysis.yml

```diff
@@ -0,0 +1,190 @@
+parameters:
+  language: ''
+  lib: false
+  skipBinSkim: false # Tells the analysis pipeline not to run binskim as it doesn't work for some samples
+  deleteBinSkim: '' # Some DLLs cause BinSkim errors and need to be removed before running
+  skipPolaris: false # Skips polaris for samples that have no realy code to analysze
+  useRequirementsTxt: false # Tell the pipeline to use requirements.txt even though the language is Jupyter instead of Python
```

Comment:
I'd maybe be more generic, like `even if the language is not Python, e.g. Jupyter`. 

We could also probably just delete this parameter, and change the checks below to check for either python or jupyter, instead of only python.

Reply:
i haven't read everything, so this comment might be off base, but you can have other languages than python in Jupyter (like R or even dotnet core)